Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '3.0.2'
+ruby '3.0.0'
```

Comment:
Initial project setup requires a ruby version 3.0.2 in order to avoid conflicts, please kindly update your ruby version to 3.02.